### ISSUE TITLE
No show content

### ISSUE THREAD

**Issue by blaine-O** [sander1/CBS.bundle#1]
Thanks for the update!  I installed and restarted and all and still have the issue where the content of individual shows does not show up:

![image](https://user-images.githubusercontent.com/8301331/50126719-52f2bd80-022b-11e9-9b47-6632d55787c6.png)

**Comment by sander1** [comment]
The error "There are no video sections to list right now" is from the old version of the plugin, so uninstall/remove the old CBS plugin, then place the one from this repository back. The difference should be visible in the icon (the new bundle has a small red banner in the bottom left that says "sander1"):

![CBS logo](https://raw.githubusercontent.com/sander1/CBS.bundle/master/Contents/Resources/icon-default.jpg "CBS logo")

**Comment by djantho** [comment]
i have uninstalled/ reinstalled as stated above. The add-on has the new icon as stated also. When i open CBS all menu's are listed but once i try to open a category it states " This plugin is not responding"  when you back out to the main menu all the show icons are visible? any thoughts?

**Comment by djantho** [comment]
i am still kinda new to all of this so any help would be greatly appreciate

**Comment by blaine-O** [comment]
@sander1 Thanks so much, I have got it working now!  At first, when I installed your forked plug-in it was somehow still loading the old plugin.  I tried again this evening and got your "Sander1" plug-in working.  I can now access the show content.  We watch a fair amount of CBS shows these days, thanks!

@djantho Not sure what help to offer other than to give it a minute after installing the new plug-in to load and refresh and fetch from the Internet?  Hope it works for you was well.

**Comment by ChrisRidl** [comment]
@djantho I ran into the same problem. Turns out Plex was looking in a different folder than the plug-ins folder located with my install. (Custom install directory). Hopefully this will help fix your problem as it helped with mine. https://support.plex.tv/articles/201106098-how-do-i-find-the-plug-ins-folder/  I believe the directory you have set under Settings > General > `The path where local application data is stored` dictates this.
